src/Resources/config/packages/monolog.yaml

```diff
@@ -1,7 +1,14 @@
 monolog:
-    channels: [ "payone_transaction_forward" ]
+    channels:
+        - payone
+        - payone_transaction_forward
 
     handlers:
+        payoneLogger:
+            type: rotating_file
+            path: "%kernel.logs_dir%/payone.log"
```

Comment:
Why without an environment like this `"%kernel.logs_dir%/payone_%kernel.environment%.log"`?

Reply:
good argument. Thank you!